packages/autorest.python/package.json

```diff
@@ -25,22 +26,20 @@
   },
   "homepage": "https://github.com/Azure/autorest.python/blob/main/README.md",
   "dependencies": {
-    "@autorest/system-requirements": "~1.0.2"
+    "@autorest/system-requirements": "~1.0.2",
+    "fs-extra": "~11.2.0"
   },
   "devDependencies": {
     "@microsoft.azure/autorest.testserver": "^3.3.46",
-    "typescript": "~5.1.3"
+    "typescript": "~5.1.3",
+    "@azure-tools/typespec-python": "workspace:^"
   },
   "files": [
     "autorest/**/*.py",
     "autorest/**/*.jinja2",
+    "scripts/",
     "setup.py",
-    "install.py",
-    "prepare.py",
-    "start.py",
-    "venvtools.py",
-    "run-python3.js",
     "requirements.txt",
-    "run_cadl.py"
+    "generator/"
```

Comment:
It is ignored in `.gitignore`, why added here?

Reply:
it is only for pack